golang/pkg/notify.go

```diff
@@ -0,0 +1,78 @@
+package notify
+
+import (
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+)
+
+type AlchemyWebhookEvent struct {
+	WebhookId string
+	Id        string
+	CreatedAt time.Time
+	Type      string
+	Event     map[string]interface{}
```

Comment:
Might be worth adding examples of parsing specific events in the example. `map[string]interface{}` can be annoying to work with.